.env.evonet

```diff
@@ -8,5 +8,6 @@ CORE_MASTERNODE_BLS_PRIV_KEY=
 CORE_EXTERNAL_IP=
 DPNS_CONTRACT_ID=77w8Xqn25HwJhjodrHW133aXhjuTsTv9ozQaYpSHACE3
 DPNS_TOP_LEVEL_IDENTITY=77w8Xqn25HwJhjodrHW133aXhjuTsTv9ozQaYpSHACE3
+CORE_P2P_PORT=20001
 # testnet or mainnet
 SENTINEL_NETWORK=testnet
```

Comment:
actually we could use `COMPOSE_FILE` env in Evo present in order to specify proper config files:
```
COMPOSE_FILE=docker-compose.yml:docker-compose.evo.yml
```

Reply:
Let's group `COMPOSE` envs together.